crev-lib/src/id.rs

```diff
@@ -82,18 +83,18 @@ impl LockedId {
             Self::weak_passphrase_config()
         };
 
-        let pwsalt = random_vec(32);
+        let pwsalt = random_vec(16);
```

Comment:
The salt is not used for AEAD, why make it weaker?

Reply:
Aes256SivAead required it. `32` caused assertion failure in generic array due to length mismatch.